image/descriptor.go

```diff
@@ -32,6 +33,10 @@ type descriptor struct {
 	Size      int64  `json:"size"`
 }
 
+func (d *descriptor) normalizeDigest() string {
+	return strings.Replace(d.Digest, ":", "-", -1)
+}
```

Comment:
the point now is that, unfortunately this tool is broken and it's broken since a lot now probably (https://github.com/opencontainers/image-spec/pull/165 is 28 days old why was that closed)
Anyhow I'm ok either way...